geomesa-nifi-processors/src/main/scala/org/geomesa/nifi/geo/AbstractGeoIngestProcessor.scala

```diff
@@ -212,82 +211,82 @@ abstract class AbstractGeoIngestProcessor(dataStoreProperties: Seq[PropertyDescr
       if (value == null) { Seq.empty } else {
         Seq(p.getName -> value)
       }
-    }.toMap
-    val props = configured ++ static // note: static props take precedence
-    logger.trace(s"DataStore properties: $props")
+    }
+    val props = configured.toMap ++ static // note: static props take precedence
+    logger.trace(s"DataStore properties: ${props.map { case (k, v) => s"$k -> $v" }.mkString(", ")}")
```

Comment:
Idly, I wonder what the toString of a Scala map is...  Would it be much different than this?

Reply:
it just includes `Map(foo -> bar, bar -> baz)` instead of `foo -> bar, bar -> baz`